tensorflow/tools/tf_sig_build_dockerfiles/README.md

```diff
@@ -102,19 +103,36 @@ Now let's build `tf-nightly`.
    Hub](https://hub.docker.com/r/tensorflow/build/tags). The options for the
    `master` branch are:
 
+For CUDA
+
     - `tensorflow/build:latest-python3.10`
     - `tensorflow/build:latest-python3.9`
     - `tensorflow/build:latest-python3.8`
     - `tensorflow/build:latest-python3.7`
 
+For ROCM
+
+    - `rocm/tensorflow-build:latest-python3.10`
+    - `rocm/tensorflow-build:latest-python3.9`
+    - `rocm/tensorflow-build:latest-python3.8`
+    - `rocm/tensorflow-build:latest-python3.7`
```

Comment:
Are these Centos based images?

Reply:
yes, these are built with the new manylinux dockerfile and pushed to dockerhub

Reply:
Great, Was not sure.